webapp/src/components/CreateMeetingNoteForm.tsx

```diff
@@ -0,0 +1,114 @@
+import { Alert, Snackbar, Stack, TextField } from '@mui/material';
+import CheckCircleOutlineIcon from '@mui/icons-material/CheckCircleOutline';
+import { LoadingButton } from '@mui/lab';
+import React, { FormEventHandler, useState } from 'react';
+
+import { EntityId } from '../types/api';
+interface CreateMeetingNoteFormProps {
+  agendaOwningParticipantId: EntityId;
+  meetingId: EntityId;
+  onMeetingNoteChanged?: (id: EntityId) => void;
+  sortOrder: number | undefined;
+}
+
+const CreateMeetingNoteForm = ({
+  agendaOwningParticipantId,
+  meetingId,
+  onMeetingNoteChanged,
+  sortOrder,
+}: CreateMeetingNoteFormProps) => {
+  const [isSavingMeetingNote, setIsSavingMeetingNote] = useState(false);
+  const [saveMeetingNoteError, setSaveMeetingNoteError] = useState(null);
+  const [isSuccessMessageVisible, setIsSuccessMessageVisible] = useState(false);
+  const [meetingNoteText, setMeetingNoteText] = useState('');
+  const [isSuccessIndicator, setIsSuccessIndicator] = useState(false);
+
+  const updateSaveMeetingNoteButton = () => {
```

Comment:
This function is only ever called once that I can find. Does it need to be extracted, or could it just be inlined?